dependency/client_set.go

```diff
@@ -385,6 +387,23 @@ func (c *ClientSet) CreateVaultClient(i *CreateVaultClientInput) error {
 	return nil
 }
 
+// If vault agent specifies wrap_ttl for the token it is returned as
+// a SecretWarpInfo struct marshalled into JSON instead of the normal raw
```

Comment:
`SecretWarpInfo` should be `SecretWrapInfo`

Reply:
My SciFi fandom leaking through. Thanks!